ui-performance.md

```diff
@@ -0,0 +1,470 @@
+---
```

Comment:
note the screen shots will need updating; we changed the style of the performance graphs recently for some reason.

Reply:
Also, the ribbon in the upper right now says "DEBUG" instead of "SLOW MODE".